frontend/src/tests/assignment-tests.js

```diff
@@ -1,10 +1,125 @@
-import { it } from "./utils";
+import PropTypes from "prop-types";
+import {
+    it,
+    expect,
+    beforeAll,
+    checkPropTypes,
+    assignmentPropTypes
+} from "./utils";
+import { databaseSeeder } from "./setup";
 
-// TODO: Remove eslint disable
-// eslint-disable-next-line
 export function assignmentsTests({ apiGET, apiPOST }) {
-    it.todo("get assignments for session");
-    it.todo("get assignments for position");
-    it.todo("create and delete assignment for position");
-    it.todo("update assignment dates/note/offer_override_pdf");
+    const session = databaseSeeder.seededData.session;
```

Comment:
for some reason, if I assigned the initial data here, I got the incorrect data ( without ids) not sure if that happened to you or not

Reply:
What do you mean by "assigned the initial data"?

Reply:
If you look at line 23, it is using `session.id` and the test is passing. (try doing `console.log` inside of a test instead of outside of the test)